Controllers/Playlist.controller.js

```diff
@@ -0,0 +1,116 @@
+const playlistsdb=require('../Models/playlists.model');
+const usersdb=require('../Models/users.model')
+
+module.exports.sendAllPlaylists=async (req,res)=>{
+    const {id}=req.params
+    try{
+        const {playlists}=await (await usersdb.findById(id)).execPopulate({path:'playlists',populate:({path:'videos'})});
```

Comment:
idk but vs code shows that await is not required here you can check that

Reply:
Hey Rishav the outer await is for the populate, I'm finding the item and populating in the same line